vuu-ui/packages/vuu-ui-controls/src/drag-drop/useDragDropCopy.ts

```diff
@@ -27,7 +27,7 @@ export const useDragDropCopy = ({
   );
 
   const drag = useCallback(() => undefined, []);
-  const drop = useCallback(() => undefined, []);
+  const drop = useCallback(() => ({ fromIndex: -1, toIndex: -1 }), []);
```

Comment:
Its for when we're dragging an item but will not actually remove that item from the source location - i.e. a copy rather than a move operation. Its much simpler.

And yes, should really sue same Literal, have moved it into useDragDropCopy and share it with useDragDrop